vcf-dumper/vcf-dumper-ws/src/main/resources/application.properties

```diff
@@ -5,9 +5,6 @@
 #spring.datasource.password=
 #spring.datasource.driver-class-name=org.postgresql.Driver
```

Comment:
I don't think we need this JDBC datasource configuration either (it is simply an alternative to JNDI).